CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fix bug where running `conjurctl server` or `conjurctl account create` with
   passwords that contain `,`s sent via stdin raised an error.
   [cyberark/conjur#2159](https://github.com/cyberark/conjur/issues/2159)
+- Specify keepalive timeout for puma to be longer than proxy and load balancers
```

Comment:
can you link to an issue please?

Reply:
how does this change affect the user?

Reply:
does it fix a known bug?

Reply:
I've added links to known-related issues in the description.  The effect should be to prevent the random and spurious 502 errors frequently seen when running conjur behind a load balancer/nginx/proxy due to mismatched keepalive times, but otherwise users should not see behavioral changes.  The issue was discovered for the ConjurOps v5 deployment, but is a symptom that has been seen pretty frequently in other contexts and always chalked up to transient issues.